### ISSUE TITLE
What is the `W` in critic loss?

### ISSUE THREAD

**Issue by fzyzcjy** [hiwonjoon/cycle-gan-tf#4]
I wonder what is the `W` in the code? It seems not to be in the CycleGAN paper, nor in WGAN-GP paper.
I would appreciate it if you could answer it. Thank you very much in advance!

Appendix: some occurrence of the W:

`W_a = tf.reduce_mean(v_a_real) - tf.reduce_mean(v_a_gen)
W_b = tf.reduce_mean(v_b_real) - tf.reduce_mean(v_b_gen)
W = W_a + W_b`
`loss_c = -1.0*W + LAMBDA*GP`
`...`
